recipes/cjson/all/conanfile.py

```diff
@@ -44,6 +45,8 @@ def configure(self):
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
+        if self.settings.compiler == "Visual Studio" and self.options.shared and self.settings.compiler.runtime == "MTd":
```

Comment:
I think you can skip MT too.
If MTd fails hard, then MT will fail subtly.

Reply:
MT doesn't fail

Reply:
The main difference between MT and MTd are the debug assertions. The fact that MTd asserts, means that MT will fail too.

Otoh, libffi MTd fails too and we keep MT available.